src/pages/contributing/component-checklist/index.mdx

```diff
@@ -0,0 +1,157 @@
+| -------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **API guiding principles** | <UnorderedList><ListItem> Prioritize end user: put the consumer/dev experience above how difficult it might be for us to implement.</ListItem><ListItem> Interoperability: support wide band coverage of multiple react versions, node LTS versions, etc. and prefer framework/library agnostic approaches to component APIs and designs.</ListItem><ListItem>Stability: always deprecate before removal, ensure long deprecation periods, consider the migration experience for developers. </ListItem><ListItem> Composition: components should be broken down into logical pieces to support the many disparate configurations that are required by the business. </ListItem><ListItem> Developer Experience: consider how a developer will use this component; will they find it approachable, confusing, complex, too simple? </ListItem></UnorderedList> | Careful consideration of API design ensures that future changes to the system can be made in an iterative way that minimize disruption. Developer experience and productivity can be heavily influenced by confusing or needlessly complex APIs. The business has a wide range of requirements and should meet as many of them as possible. Prioritize the user wherever possible - if we can make things easier for them by writing extra code, handling complexity internally, or taking care of a common concern, do so. It’s our job to contain chaos so that it’s easier and faster to build excellent software products. |
```

Comment:
```suggestion
| **API guiding principles** | <UnorderedList><ListItem> Prioritize end user: put the consumer/dev experience above how difficult it might be for us to implement.</ListItem><ListItem> Interoperability: support wide band coverage of multiple react versions, node LTS versions, etc. and prefer framework/library agnostic approaches to component APIs and designs.</ListItem><ListItem>Stability: always deprecate before removal, ensure long deprecation periods, consider the migration experience for developers. </ListItem><ListItem> Composition: components should be broken down into logical pieces to support the many disparate configurations that are required by the business. </ListItem><ListItem> Developer experience: consider how a developer will use this component; will they find it approachable, confusing, complex, too simple? </ListItem></UnorderedList> | Careful consideration of API design ensures that future changes to the system can be made in an iterative way that minimize disruption. Developer experience and productivity can be heavily influenced by confusing or needlessly complex APIs. The business has a wide range of requirements and should meet as many of them as possible. Prioritize the user wherever possible - if we can make things easier for them by writing extra code, handling complexity internally, or taking care of a common concern, do so. It’s our job to contain chaos so that it’s easier and faster to build excellent software products. |
```

Reply:
@alina-jacob What did you change here? It's not highlighting the difference and I couldn't tell.

Reply:
Suggested the 'e' of experience to be sentence case. 
Not sure why it didn't get highlighted, maybe because its not an addition / alphabet change. 
<img width="795" alt="image" src="https://github.com/user-attachments/assets/7f524e00-8180-4b03-bf62-934a1b4b2513">